### PR TITLE
Enable nullability in CursorEditor

### PR DESCRIPTION
**Pull request by halgab**
## Proposed changes

- Enable nullability in `CursorEditor`

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/9608)

### CODE REVIEW

File: src/System.Windows.Forms.Design/src/System/Drawing/Design/CursorEditor.CursorUI.cs [hunk 1]
```diff
@@ -36,26 +33,26 @@ public CursorUI()
...
                 }
             }
         }
 
-        public object Value => _value;
+        public object? Value { get; private set; }
 
         public void End()
         {
             _editorService = null;
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Design/src/System/Drawing/Design/CursorEditor.CursorUI.cs:47]: i would change this assignment as `_editorService = null!` and make `_editorService` not nullable.

Comment (halgab) [reply src/System.Windows.Forms.Design/src/System/Drawing/Design/CursorEditor.CursorUI.cs:47]: `_editorService` is not set in the constructor though, so I would need to change line 18 to:

> `private IWindowsFormsEditorService _editorService = null!;`

Should I make the change anyway?

Comment (dreddy-work) [reply src/System.Windows.Forms.Design/src/System/Drawing/Design/CursorEditor.CursorUI.cs:47]: hmm. its assigned in `Start`. That doesn't help. lets keep as-in now.

**Review by dreddy-work — APPROVED** [review]

**Comment by dreddy-work** [comment]
Please resolve conflict.

**Comment by halgab** [comment]
Should be good now

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**